src/components/SystemsPage/SystemsPageRow.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
+import {
+  Box,
+  Collapse,
+  IconButton,
+  makeStyles,
+  TableCell,
+  TableRow,
+} from '@material-ui/core';
+import KeyboardArrowDownIcon from '@material-ui/icons/KeyboardArrowDown';
+import KeyboardArrowUpIcon from '@material-ui/icons/KeyboardArrowUp';
+import { parseEntityName } from '@backstage/catalog-model';
+
+import { Gauge } from '../Gauge/Gauge';
+import { defaultSystemRefContext } from '../../utils/ComponentUtils';
+import { SystemsPageRowDetails } from './SystemsPageRowDetails';
+
+import { AverageScoresByIdentifier } from '../../api/types';
+
+const useStyles = makeStyles({
+  root: {
+    '& > *': {
+      borderBottom: 'unset!important',
+    },
+    height: '35px',
+  },
+  openIcon: {
+    width: '35px',
+  },
+  progress: {
+    height: '64px',
+    marginRight: '16px',
+  },
+});
+
+interface Props {
+  serviceScore: AverageScoresByIdentifier;
+}
+
+export const SystemsPageRow: React.FC<Props> = ({ serviceScore }) => {
+  const [open, setOpen] = useState(false);
+  const classes = useStyles();
+  const { name } = parseEntityName(
+    serviceScore?.identifier ?? '',
+    defaultSystemRefContext,
```

Comment:
this one we should use the `defaultComponentRefContext`, since these entity names are all "components" (aka services in cortex)